packages/mongo-nav/src/mongo-select/MongoSelect.tsx

```diff
@@ -89,140 +104,137 @@ const onKeyDown: React.KeyboardEventHandler = e => {
   }
 };
 
-function MongoSelect(
-  props: ProjectMongoSelectProps | OrganizationMongoSelectProps,
-) {
-  const {
-    onClick,
-    variant,
-    onChange,
-    className,
-    urls,
-    isActive = false,
-  } = props;
-
-  let trigger, footer;
-  const { mongoSelect } = urls;
-
-  if (isProject(props)) {
-    trigger = <ProjectTrigger current={props.current} className={className} />;
-    footer = (
-      <li onKeyDown={onKeyDown} role="none" className={projectButtonStyle}>
-        <FocusableMenuItem>
-          <Button href={mongoSelect.viewAllProjects} as="a">
-            View All Projects
-          </Button>
-        </FocusableMenuItem>
-        <FocusableMenuItem>
-          <Button href={mongoSelect.newProject} as="a">
-            + New Project
-          </Button>
-        </FocusableMenuItem>
-      </li>
-    );
-  } else {
-    trigger = (
-      <OrganizationTrigger
-        current={props.current}
-        className={className}
-        urls={urls}
-        isActive={isActive}
-      />
-    );
-    footer = (
-      <MenuItem onKeyDown={onKeyDown} href={mongoSelect?.viewAllOrganizations}>
-        <strong className={viewAllStyle}>View All Organizations</strong>
-      </MenuItem>
-    );
-  }
-
-  const renderProjectOption = (
-    props: ProjectMongoSelectProps,
-    datum: ProjectInterface,
-  ) => {
-    const content = (
-      <ProjectOption
-        projectName={datum.projectName}
-        href={props.constructProjectURL(datum.orgId, datum.projectId)}
-      />
-    );
+function OrgSelect({
+  current,
+  data,
+  className,
+  urls,
+  isActive,
+  onChange,
+  onClick,
+  constructOrganizationURL,
+}: OrganizationMongoSelectProps) {
+  const renderOrganizationOption = (datum: OrganizationInterface) => {
+    const { orgId, orgName, planType } = datum;
 
     return (
       <MenuItem
-        key={datum.projectId}
+        key={orgId}
         className={menuItemContainerStyle}
         onClick={onClick}
+        href={constructOrganizationURL(orgId)}
       >
-        {content}
+        <div className={orgOptionContainerStyle}>
+          <span className={nameStyle}>{orgName}</span>
+          <span className={productStyle}>{formatPlanType(planType)}</span>
+        </div>
       </MenuItem>
     );
   };
 
-  const renderOrganizationOption = (
-    props: OrganizationMongoSelectProps,
-    datum: OrganizationInterface,
-  ) => {
-    const content = (
-      <OrganizationOption
-        orgName={datum.orgName}
-        planType={datum.planType}
-        href={props.constructOrganizationURL(datum.orgId)}
-      />
-    );
+  const errorMessage =
+    'You do not belong to any organizations. Create an organization to start using MongoDB Cloud';
+
+  return (
+    <Menu
+      trigger={
+        <OrganizationTrigger
+          current={current?.orgName ?? 'All Organizations'}
```

Comment:
Should we just accept a `placeholder` prop, and if current is unset, render that?

Reply:
Do you think different products are going to want to change this value? That would mean exposing two props one for Orgs and one for Projects.. Idk if it's necessary